ckanext/cfpb_extrafields/templates/package/resource_read.html

```diff
@@ -105,8 +110,71 @@
               {% endif %}
               
             </tbody>
-          </table>
-        </div>
-      </section>
-    {% endif %}
+    </table>
+  </div>
+  <div class="module-content">
+<!-- ============ SHARE ============= -->
+<!-- CODE REVIEW: This must come before the VIEW section below for some reason?! -->
```

Comment:
My only guess here is about the order in which things get loaded.  It must need these resources in order to render, but I don't have any specific details.  Sorry :(